File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/chts/CHT0117Q00GrdCHT0118Info.java
package nta.med.data.model.ihis.chts;

import java.io.Serializable;

public class CHT0117Q00GrdCHT0118Info implements Serializable {
	private String buwi;
    private String subBuwi;
    private String subBuwiName;
    private String contKey;
	public CHT0117Q00GrdCHT0118Info(String buwi, String subBuwi,
			String subBuwiName, String contKey) {
		super();
		this.buwi = buwi;
		this.subBuwi = subBuwi;
		this.subBuwiName = subBuwiName;
		this.contKey = contKey;
	}
	public String getBuwi() {
		return buwi;
	}
	public void setBuwi(String buwi) {
		this.buwi = buwi;
	}
	public String getSubBuwi() {
		return subBuwi;
	}
	public void setSubBuwi(String subBuwi) {
		this.subBuwi = subBuwi;
	}
	public String getSubBuwiName() {
		return subBuwiName;
	}
	public void setSubBuwiName(String subBuwiName) {
		this.subBuwiName = subBuwiName;
	}
	public String getContKey() {
		return contKey;
	}
	public void setContKey(String contKey) {
		this.contKey = contKey;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Chts/CHT0117Q00GrdCHT0118Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Chts
{
    [Serializable]
	public class CHT0117Q00GrdCHT0118Info
	{
		private String _buwi;
		private String _subBuwi;
		private String _subBuwiName;
		private String _contKey;

		public String Buwi
		{
			get { return this._buwi; }
			set { this._buwi = value; }
		}

		public String SubBuwi
		{
			get { return this._subBuwi; }
			set { this._subBuwi = value; }
		}

		public String SubBuwiName
		{
			get { return this._subBuwiName; }
			set { this._subBuwiName = value; }
		}

		public String ContKey
		{
			get { return this._contKey; }
			set { this._contKey = value; }
		}

		public CHT0117Q00GrdCHT0118Info() { }

		public CHT0117Q00GrdCHT0118Info(String buwi, String subBuwi, String subBuwiName, String contKey)
		{
			this._buwi = buwi;
			this._subBuwi = subBuwi;
			this._subBuwiName = subBuwiName;
			this._contKey = contKey;
		}

	}
}